java/core/src/main/java/co/worklytics/psoxy/HashUtils.java

```diff
@@ -1,11 +1,14 @@
 package co.worklytics.psoxy;
 
+import lombok.NoArgsConstructor;
 import org.apache.commons.codec.digest.DigestUtils;
 import org.apache.commons.lang3.StringUtils;
 
+import javax.inject.Inject;
 import java.nio.charset.StandardCharsets;
 import java.util.Base64;
 
+@NoArgsConstructor(onConstructor = @__({@Inject}))
```

Comment:
why not `@NoArgsConstructor(onConstructor_ = @Inject)` ? cleaner imho

Reply:
yeah, is confusing that Lombok's doc still points to `@__({@Inject}))`
> To put annotations on the generated constructor, you can use onConstructor=@__({@AnnotationsHere}), but be careful; this is an experimental feature. For more details see the documentation on the onX feature.

But yes, that's JDK7 style, JDK8 way cleaner. I always get confused about what's what